diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/smoke/DiktatSmokeTest.kt

```diff
@@ -342,6 +342,12 @@ class DiktatSmokeTest : FixTestBase("test/smoke/src/main/kotlin",
             }
     }
 
+    @Test
+    @Tag("DiktatRuleSetProvider")
+    fun `fix can cause long line`() {
```

Comment:
why this test is here? I guess LongLine rule has a dedicated file\class for it

Reply:
Because the problem is precisely in the use of several rules. 

As I commented in the issue:
https://github.com/saveourtool/diktat/issues/1339#issuecomment-1157808635